android/app/src/main/java/com/poweroutrage/MainActivity.java

```diff
@@ -15,8 +15,8 @@ protected String getMainComponentName() {
     return "PowerOutrage";
   }
   @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        SplashScreen.show(this, true);
-        super.onCreate(savedInstanceState);
-    }
+  protected void onCreate(Bundle savedInstanceState) {
+    SplashScreen.show(this, true);
+      super.onCreate(null);
```

Comment:
@usaidpeerzada is there a reason you are not passing in the `savedInstanceState`?

Reply:
https://reactnavigation.org/docs/getting-started